Ref/reorder readme (#4)
edgelesssys/graphene-tensorflow-demo: README.md

pipe install grpcio~=1.34.0
```

## Running the demo

We provide [a docker image](https://github.com/orgs/edgelesssys/packages/container/package/tensorflow-graphene-marble) to run TensorFlow Serving with Graphene and Marblerun.

> However you may also build your own by following the steps [of this tutorial](#Building-the-Docker-Image).

Reviewer: ```suggestion
You can also [build it yourself](#Building-the-Docker-Image).
```
short sentences

> Note that, if you plan to build your own image, you will have to change the image name in `kubernetes/templates/tf-server.yaml`.

Reviewer: maybe move the sentence about renaming into the corresponding section?

> ### As a standalone
> 
> You can run the demo with Marblerun as a standalone by following these steps.

Reviewer: ```suggestion
### Standalone

You can run the demo with Marblerun in standalone mode as follows:
```

    ```

1. Start the Tensorflow Model Server

>     This will pull our docker image. If you wish to use your own change `image_id` in `tools/run_tf_image.sh` to the name of your image.

Reviewer: parameterize the script instead?
